src/test/java/seedu/sellsavvy/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,7 @@ public class TypicalIndexes {
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_PERSON = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_PERSON = Index.fromOneBased(3);
+    public static final Index INDEX_FIRST_ORDER = Index.fromOneBased(1);
```

Comment:
maybe instead of adding the same thing change on top to INDEX_FIRST instead of INDEX_FIRST_PERSON?

Reply:
Agreed, this indexing can be generalised to both persons and orders.

Reply:
I think it should be separated for code clarity; you had a comment earlier asking me to change INDEX to ORDER_INDEX to differentiate from PERSON_INDEX in the DeleteOrderCommand

Reply:
I will resolve this and open a new issue about it